remote-access/samba.md

```diff
@@ -80,8 +80,13 @@ Firstly, create a folder to share. This example creates a folder called `shared`
 ```bash
 cd ~
 mkdir shared
+chmod 0740 share
 ```
+Now we need to tell Samba there is a 'pi'user when accessing that folder. When asked enter the passwd of pi - ether default or hopefully your own security password
```

Comment:
ether -> either ?

Reply:
Rewritten that line, but still not entirely happy with it.

Reply:
AFAICT the password used by [smbpasswd](https://www.samba.org/samba/docs/current/man-html/smbpasswd.8.html) is actually different to the password used to login to Raspbian. I _think_ the OP was trying to say something like "you can use the same password for both your regular user account and your samba account, but it's more secure if you choose a separate password for samba".

Of course I suspect that in practice 90% of users _will_ use the same password for both :wink:

Reply:
Feel free to change as you see fit. Although I wrote the original page, I am not particular familiar with this stuff.